atspi-common/src/events/keyboard.rs

```diff
@@ -72,11 +74,17 @@ impl BusProperties for ModifiersEvent {
 	const MATCH_RULE_STRING: &'static str =
 		"type='signal',interface='org.a11y.atspi.Event.Keyboard',member='Modifiers'";
 	const REGISTRY_EVENT_STRING: &'static str = "Keyboard:";
+}
 
+#[cfg(feature = "zbus")]
+impl MessageConversion for ModifiersEvent {
 	type Body = EventBodyOwned;
 
-	fn from_message_parts(item: ObjectRef, body: Self::Body) -> Result<Self, AtspiError> {
-		Ok(Self { item, previous_modifiers: body.detail1, current_modifiers: body.detail2 })
+	fn try_from_message_unchecked(msg: &zbus::Message) -> Result<Self, AtspiError> {
```

Comment:
What I see [usually](https://doc.rust-lang.org/stable/std/?search=unchecked) is that the function postfix 'unchecked' is used in Rust to indicate the function has no guardrails. For me, it is unexpected to return a `Result<T>` here. 

The function does have some guardrails, but not all - it does not check for the matching message. Maybe consider calling it 'unverified' or 'unmatched'? 

This function may also be `unsafe` because I don't think all `Message` `Body`s on the bus would lead to valid bodies of `Self`, right? 

 Maybe you have given this a bunch more thought than I have, so I'd like to hear your thoughts.

Reply:
I am basing this off of [`zbus::Message::Body::deserialized_unchecked`](https://docs.rs/zbus/latest/zbus/message/struct.Body.html#method.deserialize_unchecked); although `zbus` using something does not mean that it's necessarily correct. And you're right that this is not really the expected API here.

Perhaps there is a better name for it... I just don't know what that is. We only check that the body deserializes to the right type, and we do not check that either the signature of the body matches, neither that the interface or member matches.

Reply:
Renamed to `try_from_validated_message`